citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ConceptpowerImporter.java

```diff
@@ -0,0 +1,158 @@
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.LaxRedirectStrategy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import edu.asu.diging.citesphere.core.authority.IImportedAuthority;
+import edu.asu.diging.citesphere.core.exceptions.AuthorityServiceConnectionException;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.authority.impl.AuthorityEntry;
+import edu.asu.diging.citesphere.web.user.AuthoritySearchResult;
+
+@Component
+@PropertySource(value = "classpath:/config.properties")
+public class ConceptpowerImporter extends BaseAuthorityImporter {
+
+    private final String ID = "authority.importer.conceptpower";
+
+    @Value("${_conceptpower_authority_search_keyword}")
+    private String conceptpowerSearchKeyword;
+
+    @Value("${_conceptpower_url}")
+    private String conceptpowerURL;
+
+    private RestTemplate restTemplate;
+
+    @PostConstruct
+    private void postConstruct() {
+
+        HttpComponentsClientHttpRequestFactory factory = new HttpComponentsClientHttpRequestFactory();
+        HttpClient httpClient = HttpClientBuilder.create().setRedirectStrategy(new LaxRedirectStrategy()).build();
+        factory.setHttpClient(httpClient);
+
+        restTemplate = new RestTemplate();
+        restTemplate.setRequestFactory(factory);
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * edu.asu.diging.citesphere.authority.impl.AuthorityImporter#isResponsible(java
+     * .lang.String)
+     */
+    @Override
+    public boolean isResponsible(String source) {
+
+        return false;
+    }
+
+    @Override
+    public boolean isResponsibleForSearch(String source) {
+        if (source.equals(CONCEPTPOWER)) {
+            return true;
+        }
+        return false;
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.diging.citesphere.authority.impl.AuthorityImporter#
+     * retrieveAuthorityData(java.lang.String)
+     */
+    @Override
+    public IImportedAuthority retrieveAuthorityData(String searchString)
+            throws URISyntaxException, AuthorityServiceConnectionException {
+        return null;
```

Comment:
let's throw a not implemented exception here (either create your own runtime exception or throw a java.lang.UnsupportedOperationException).